kagami/webui/src/api/api.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL:  'http://127.0.0.1:4523/m1/5454758-5129914-default',
```

Comment:
You can use Vite's feature to statically inject backend's baseURL. Ref: https://vite.dev/guide/env-and-mode.html